packages/delayedInbox-l2msg/scripts/withdrawFunds.js

```diff
@@ -0,0 +1,147 @@
+const { providers, Wallet } = require('ethers')
+const hre = require('hardhat')
+const ethers = require('ethers')
+const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC', 'L1RPC', 'INBOX_ADDR'])
```

Comment:
Here too (We could use SDK to load the Inbox address instead gf having users to set it as an env var)